src/main/resources/application.conf

```diff
@@ -28,6 +30,9 @@ akka {
       native = off
       dir = "target/shared-journal"
     }
-    snapshot-store.local.dir = "target/snapshots"
+    snapshot-store {
```

Comment:
would be nice to make the `journal` section also nested like you did here with `snapshot-store` for consistency then.